src/main/java/com/google/codeu/data/Post.java

```diff
@@ -15,22 +15,36 @@
 	private UUID postID;
 	private String creator;
 	private long timestamp;
-	private String coverInamgeUrl;
+	private String coverImageUrl;
+	private String title;
+	private String content;
 	private LocationMarker marker;
 	private List<ChartDataRow> chartData;
 	private List<Message> messages;
 
-	public Post(String creator, String coverImageUrl, LocationMarker marker, List<ChartDataRow> chartData,
-			List<Message> messages) {
-		this(UUID.randomUUID(), creator, System.currentTimeMillis(), coverImageUrl, marker, chartData, messages);
+	public Post(String creator, String coverImageUrl, String title, String content, LocationMarker marker,
+			List<ChartDataRow> chartData, List<Message> messages) {
+		this(UUID.randomUUID(), creator, System.currentTimeMillis(), coverImageUrl, title, content, marker, chartData,
+				messages);
 	}
 
-	public Post(UUID postID, String creator, long timestamp, String coverImageUrl, LocationMarker marker,
-			List<ChartDataRow> chartData, List<Message> messages) {
+	public Post(UUID postID, String creator, long timestamp, String coverImageUrl, String title, String content) {
+		this(postID, creator, timestamp, coverImageUrl, title, content, null, null, null);
+	}
+
+	public Post(UUID postID, String creator, long timestamp, String coverImageUrl, String title, String content,

```

Comment:
I think we want this one but without the UUID postID, this is the one that should use UUID.randonUUID().

Reply:
Can we please just keep it for now? I'm like really tired. it's midnight here. These seem like little things that don't really need to be changed.